src/components/interfaces/HMI_API.xml

```diff
@@ -5441,6 +5468,9 @@
     <param name="menuIcon" type="Common.Image" mandatory="false">
       <description>The image field for AddSubMenu</description>
     </param>
+    <param name="secondaryImage" type="Common.Image" mandatory="false">
```

Comment:
Similar to the `AddCommand` case, it seems correct not to add `secondaryText` and `tertiaryText` here but it doesn't match https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0267-addcommand-ui-updates.md#addsubmenu-1.

Reply:
Resolved by https://github.com/smartdevicelink/sdl_evolution/pull/1111